### PR TITLE
[MM-29944] Add plugin icon and metadata

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
This PR adds a plugin icon that is shown in the Marketplace modal

### Bright theme
Screenshot from 2020-10-23 13-07-13

### Dark theme
Screenshot from 2020-10-23 11-29-47

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-29944

### CODE REVIEW

**Review by abhijit-singh — APPROVED** [review]
Looks good for now.

File: plugin.json [hunk 1]
```diff
@@ -2,6 +2,10 @@
     "id": "com.mattermost.plugin-incident-response",
     "name": "Incident Response",
     "description": "This plugin allows users to coordinate and manage incidents within Mattermost.",
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-incident-response/",
+    "support_url": "https://github.com/mattermost/mattermost-plugin-incident-response//issues",
```
Comment (crspeller) [inline plugin.json:6]:
```suggestion
    "support_url": "https://github.com/mattermost/mattermost-plugin-incident-response/issues",
```
(typo on the last slash)

**Review by crspeller — APPROVED** [review]
Pre-approved, one typo.

File: plugin.json (hunk 1 above)
Comment (hanzei) [reply plugin.json]: Good catch :+1:

**Review by agarciamontoro — APPROVED** [review]
We still need to review the icon in the circle, but that can wait. Looks good to me!

**Comment by hanzei** [comment]
@itao What do you think about this change?

**Comment by itao** [comment]
Hmm I would've expected the dark theme icon to have inverse colours rather than a white circle. @hanzei Is showing a different icon file in dark theme supported? And if so, @abhijit-singh could you help us make an inverse version?

**Comment by hanzei** [comment]
> Hmm I would've expected the dark theme icon to have inverse colours rather than a white circle. @hanzei Is showing a different icon file in dark theme supported? And if so, @abhijit-singh could you help us make an inverse version?

@itao Good point. Using a white circle vs inverting the colours was discussed when the Marketplace was introduced and the decision was to use the white circle for every icon as for themes that are between bright and dark it was hard to properly invert the colors. Therefore showing a different icon for different themes is currently not supported.

Please note that the "white circle" approach is used in other areas as well like the slash command icon.

**Comment by itao** [comment]
@hanzei Ah gotcha, thanks for explaining that to me.

**Review by itao — APPROVED** [review]

**Pull request merged**